### ISSUE TITLE
Takes far too long to update

### ISSUE THREAD

**Issue by Laureeeeeeeeeeeeeeen** [dlrudie/Snip#538]
I am looking at update times of 1 to 2 minutes. On a ~2 minute song it's basically useless.
Can anything be done about this?

**Comment by karkkijokeri** [comment]
I also have this problem, I would suggest that they add a customized update timer

**Comment by karabineq** [comment]
yeah i also have the problem, it sometime work normally and sometime after like two minutes

**Comment by Conholius** [comment]
Same problem and haven't found a solution

**Comment by edwrios** [comment]
same problem here

**Comment by jmswrnr** [comment]
If you're using it with Spotify, I've created an alternative project that uses a different method and updates instantly: https://github.com/jmswrnr/spotilocal

**Comment by Conholius** [comment]
Any chance to integrate [this design](`https://github.com/adarhef/NowPlaying) to Spotilocal? I already made a issue thread on the github page because I love the simplified design and animations.

I already tried to make it work with 0 programming skills and of course it didn't work :D
